packages/block-library/src/cover/edit/inspector-controls.js

```diff
@@ -73,13 +73,14 @@ function CoverHeightInput( {
 
 	return (
 		<UnitControl
+			__next40pxDefaultSize
 			label={ __( 'Minimum height of cover' ) }
 			id={ inputId }
 			isResetValueOnUnitChange
 			min={ min }
 			onChange={ handleOnChange }
 			onUnitChange={ onUnitChange }
-			__unstableInputWidth="80px"
+			__unstableInputWidth="116px"
```

Comment:
| Before | After |
|--------|-------|
|<img src="https://github.com/user-attachments/assets/5241196e-b704-4fa0-80e3-b87217e5fb26" alt="Cover block inspector, before" width="264">|<img src="https://github.com/user-attachments/assets/6a87ba90-583a-402c-89cb-e679727b884a" alt="Cover block inspector, after" width="264">|

This custom styling that allows the label to expand wider than the control is... not great. I agree that the visual UI looks the best this way in this specific instance, but I struggle to think of a universally nice way to capture this pattern. The custom styling is applied from an English-centric eye, so it's likely that we're not doing this for UnitControls that happen to have a label that doesn't overflow in English. And what if we ever change the font size or styling of the labels, and some labels that previously fit in a single column start to overflow?

Would it be worth making this label-expanding the default behavior for "single-column" controls that have enough space for the label to expand (not sure if even feasible)?

Other more simple guidelines to maintain consistency would be:

- Don't do single-column when there is nothing in the second column
    <img src="https://github.com/user-attachments/assets/507696fd-9ee6-4d35-bfa6-9be181cda30c" alt="Full-width UnitControl" width="264">

- Don't do the label expanding when single-column
    <img src="https://github.com/user-attachments/assets/dae14ddc-f370-4b77-89e4-a2db9fd08fc3" alt="Single-column UnitControl with no label expanding" width="264">

Thoughts? @WordPress/gutenberg-design

Reply:
I personally think "Min. height" would be okay, though this question has come up in the past and suggested that "Minimum height" is better.

But can it be a full-row, with input on the left, and slider on the right? This would only work when minimum height is the only control, where for example "Width" and "Height" make a good two-column pair.

Reply:
Ok, I think I can do "Minimum height" for this one. I guess the universal guideline remains that these labels should ideally fit within a single column, and no cheating just because it happens to overflow in English 😂

> But can it be a full-row, with input on the left, and slider on the right?

This has come up before (#60633), but there's some disagreement as to whether sliders are appropriate for inputs without a clear upper bound, as is in this case.

Reply:
Tweaked "After":

<img width="258" alt="Cover block inspector" src="https://github.com/user-attachments/assets/a9e6575d-65a7-4ab3-a87a-56acf433a3ea">

Reply:
This is something that should be solved universally, and not on a case-by-case basis, especially since we can't predict the length of those labels when they get translated to other languages:

- we should keep our strings as short and clear as possible;
- we should have a universal solution for when these input label don't fit in their space — could we truncate + ellipse them?

Reply:
They are truncated + ellipsized by default. It's just that this case was initially "cheating" by expanding only the label into two columns.

Reply:
I see — it would be probably be better to remove the "cheat", if possible

Reply:
Yes we did remove it, this is the [before/after](https://github.com/WordPress/gutenberg/pull/64520/files#r1718581029).